### ISSUE TITLE
BMP 280

### ISSUE THREAD

**Issue by haxord7hasib** [htgdokania/Drone_altitude_hold-#1]
Hello bro, is it possible to use BMP280 Barometer sensor for altitude hold instead of ultrasonic sensor

**Comment by htgdokania** [comment]
Yes initially I tried using bmp280 only...but the issue was , I was not
getting proper altitude data using this...if you are  able to somehow
figure out how to get altitude with good accuracy and that too in very less
time then we can use this here.

Since it was taking too much time to figure out bmp280 I ended up using an
ultrasonic sensor which is extremely easy to use and has good accuracy.
